src/hotspot/cpu/x86/x86.ad

```diff
@@ -10468,3 +10485,17 @@ instruct DoubleClassCheck_reg_reg_vfpclass(rRegI dst, regD src, kReg ktmp, rFlag
   %}
   ins_pipe(pipe_slow);
 %}
+
+
+instruct selectFromTwoVec_evex(vec dst, vec src1, vec src2)
```

Comment:
You could rename `dst` -> `mask_and_dst`. That would maybe help the reader to more quickly know that it is an input-mask and output-dst.